ansible_collections/arista/avd/roles/eos_designs/doc/fabric-topology.md

```diff
@@ -215,6 +215,13 @@ defaults <- node_group <- node_group.node <- node
     # Point-to-Point interface speed - will apply to uplinks on both ends | Optional.
     uplink_interface_speed: < interface_speed | forced interface_speed | auto interface_speed >
 
+    # Custom structured config applied to "uplink_interfaces", and "uplink_switch_interfaces"
+    # When uplink_type == "p2p", custom structured config added under ethernet_interfaces.<interface> for eos_cli_config_gen
+    # Overrides the settings on the ethernet interface level.
+    # When uplink_type == "port-channel", custom structured config added under port_channel_interfaces.<interface> for eos_cli_config_gen
+    # Overrides the settings on the port-channel interface level.
+    uplink_structured_config: < dictionary >
```

Comment:
What is the precedence expectation if there is a default, a node_group or node level `structured_config`?


https://github.com/aristanetworks/ansible-avd/pull/1865/files#diff-0f532e75adc87de333555aa88096862c6a58b61fec91adbf5e76055b3a814140R181

Reply:
Good point, let's add a test case, and update the documentation accordingly!

Reply:
added a suggestion